src/drivers/i2c/adapters/imx_i2c/imx_i2c1.c

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
Why it's placed in `src/drivers/i2c/adapters/imx_i2c/imx_i2c1.c`, not `src/drivers/i2c/imx/imx_i2c1.c`?